Makefile

```diff
@@ -25,25 +39,18 @@ default: docs
 .PHONY: docs
 docs: output/docs.pdf output/docs.html
 
+ifeq "$(strip $(PANDOC))" ''
+output/docs.pdf output/docs.html:
```

Comment:
lets do that in another PR. I have mulled on a better name and a way to do include the version (git tag + abbreviated commit?) but haven't arrived at a clean enough conclusion yet.